packages/ui-core/src/components/Counter/Counter.test.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import Counter from './Counter';
+
+const props = {
+    className: 'test-class',
+    initialValue: 10,
+    min: 3,
+    max: 33,
+    isDisabled: true,
+};
+
+describe('<Counter />', () => {
+    describe('layout', () => {
+        it('renders Counter', () => {
+            const wrapper = shallow(
+                <Counter />
+            );
+            expect(wrapper).toMatchSnapshot();
+        });
+
+        it('renders Counter with props', () => {
+            const wrapper = shallow(
+                <Counter {...props} />
+            );
+            expect(wrapper).toMatchSnapshot();
+        });
+    });
+
+    describe('handlers', () => {
+        it('calls onClick for buttons', () => {
```

Comment:
Проверяем, что вызывается колбэк, т.е. 
```suggestion
        it('calls onChange for buttons', () => {
```

Ниже везде тоже самое.